test/clj_sqs_extended/core_test.clj

```diff
@@ -8,52 +8,67 @@
 
 (use-fixtures :once fixtures/with-localstack-environment)
 
-(def ^:private test-message-1
-  {:quote "State. You're doing it wrong."
-   :by "Rich Hickey"})
-
-(def ^:private test-message-2
-  {:quote "Nothing says 'Screw You!' like a DSL."
-   :by "Stuart Halloway"})
+(def ^:private test-messages
+  [{:quote "State. You're doing it wrong."
+    :by    "Rich Hickey"}
+   {:quote "Nothing says 'Screw You!' like a DSL."
+    :by    "Stuart Halloway"}
+   {:quote "I don't care if it works on your machine! We are not shipping your machine!"
+    :by    "Vidiu Platon"}
+   {:quote "If you can't beat your computer at chess, try kickboxing."
+    :by    "Claudio E. Montero"}
+   {:quote "I would rather be without a state than without a voice."
+    :by    "Edward Snowden"}])
 
 (def ^:private test-message-larger-than-256kb
   (helpers/random-string-with-length 300000))
 
 (deftest can-receive-message
   (testing "Sending/Receiving basic maps"
     (let [client (fixtures/localstack-sqs)
-          url (fixtures/test-queue-url)]
+          url (fixtures/test-queue-url)
+          test-message (first test-messages)]
       (doseq [format [:transit :json]]
-        (sqs-ext/send-message client url test-message-1 {:format format})
+        (sqs-ext/send-message client url test-message {:format format})
         (let [message (sqs-ext/receive-message client url {:format format})]
-          (is (= test-message-1 (:body message))))))))
+          (is (= test-message (:body message))))))))
 
 (deftest can-auto-delete-message
   (testing "Auto-Deleting a single message after receiving it"
     (let [client (fixtures/localstack-sqs)
-          url (fixtures/test-queue-url)]
+          url (fixtures/test-queue-url)
+          test-message (rand-nth test-messages)]
```

Comment:
good re-use of `test-messages`. I would statically pick one so that there's no randomness in this test that doesn't need it. For repeatability's sake.